actors/migration/test/correct_cc_upgrade_then_fault_scenario_test.go

```diff
@@ -394,19 +396,13 @@ func TestMigrationCorrectsCCThenFaultIssue(t *testing.T) {
 	err = v2.GetState(minerAddrs.IDAddress, &st2)
 	require.NoError(t, err)
 
-	act, found, err = v2.GetActor(minerAddrs.IDAddress)
+	stateTree, err := v2.GetStateTree()
 	require.NoError(t, err)
-	require.True(t, found)
-
-	_, acc, err = miner2.CheckStateInvariants(&st2, v2.Store(), act.Balance)
+	totalBalance, err := v2.GetTotalActorBalance()
 	require.NoError(t, err)
-
-	assert.True(t, acc.IsEmpty())
-	if !acc.IsEmpty() {
-		for _, m := range acc.Messages() {
-			fmt.Println(m)
-		}
-	}
+	acc, err = states.CheckStateInvariants(stateTree, totalBalance, v2.GetEpoch())
+	require.NoError(t, err)
+	assert.True(t, acc.IsEmpty(), strings.Join(acc.Messages(), "\n"))
```

Comment:
Upon reverting the migration fix above, I guess this won't hold. It's important that the state checks are useful to us as human network operators, at least as much as they are useful for tests. After the upgrade, we need to be able to run the state checks and see the un-fixed mismatch. That means this test must tolerate non-empty errors.

I think change this to assert the expected error message/s are returned.